recipes/hap.py/meta.yaml

```diff
@@ -35,6 +35,7 @@ requirements:
     - htslib
     - bcftools
     - samtools
+    - rtg-tools
```

Comment:
Maybe you should add here `- boost-cpp`

Reply:
Thanks I'll try that :)

Reply:
It's already in there (line 34)

Reply:
Ah sorry, I didn't saw the whole file

Reply:
No problem! Still thanks for the suggestions!